### PR TITLE
Fix paths / imports in sphinx configuration

### PR DESCRIPTION
**Pull request by FFroehlich**

### CODE REVIEW

File: documentation/conf.py [hunk 1]
```diff
@@ -446,7 +439,7 @@ def skip_member(app, what, name, obj, skip, options):
 
     return None
 
-
+version
```
Comment (dweindl) [inline documentation/conf.py:442]:
```suggestion

```

File: documentation/conf.py [hunk 2]
```diff
@@ -260,9 +278,8 @@
 
 
 def process_docstring(app, what, name, obj, options, lines):
-
     # only apply in the amici.amici module
-    if len(name.split('.')) < 2 or name.split('.')[1] != 'amici':
+    if len(name.split('.')) < 2 or name.split('.')[1] not in ['amici', 'amici_without_hdf5']:
```
Comment (dweindl) [inline documentation/conf.py:282]:
```suggestion
    if len(name.split('.')) < 2 or name.split('.')[1] != 'amici':
```

File: scripts/run-sphinx.sh [hunk 3]
```diff
@@ -5,9 +5,14 @@ SCRIPT_PATH=$(dirname $BASH_SOURCE)
 AMICI_PATH=$(cd $SCRIPT_PATH/.. && pwd)
 
 cd ${AMICI_PATH}/documentation
-source ../build/venv/bin/activate
-pip3 install -r ${AMICI_PATH}/documentation/rtd_requirements.txt
+python3 -m venv ${AMICI_PATH}/doc-venv --clear
+source ${AMICI_PATH}/doc-venv/bin/activate
+python -m pip install --upgrade --no-cache-dir pip
+python -m pip install git+https://github.com/readthedocs/readthedocs-sphinx-ext
+python -m pip install --upgrade --no-cache-dir Pygments==2.3.1 setuptools==41.0.1 docutils==0.14 mock==1.0.1 pillow==5.4.1 alabaster==0.7.12 commonmark==0.8.1 recommonmark==0.5.0 sphinx<2 sphinx-rtd-theme<0.5 readthedocs-sphinx-ext<1.1
```
Comment (dweindl) [inline scripts/run-sphinx.sh:12]: Move to requirements file? if leaving here, the parts with `<` / `>` will need quotes.

Comment (FFroehlich) [reply scripts/run-sphinx.sh:12]: thats all the stuff that rtd installs. I believe we can remove all of that

File: documentation/conf.py [hunk 4]
```diff
@@ -53,23 +59,49 @@
...
+    subprocess.run([
+        'python', '-m', 'pip', 'install', '--verbose', '-e',
+        os.path.join(amici_dir, 'python', 'sdist')
+    ], check=True)
+
+    from importlib import invalidate_caches
+    invalidate_caches()
+
+    sys.path.insert(0, amici_dir)
+    sys.path.insert(0, os.path.join(amici_dir, 'python', 'sdist'))
```
Comment (dweindl) [inline documentation/conf.py:83]: Don't get why they are necessary, but they are...

**Review by dweindl — APPROVED** [review]
Good enough for me...

**Pull request merged**